Handle ';' as comment char in sysctl files

alikins opened the pull request:
Fixes #20569

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 
 - Bugfix Pull Request


##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
lib/ansible/modules/system/sysctl.py

##### ANSIBLE VERSION
<!--- Paste verbatim output from “ansible --version” between quotes below -->
```
ansible 2.3.0 (sysctl_comments_20569 e35734c7d3) last updated 2017/01/23 15:31:32 (GMT -400)
  config file = /home/adrian/.ansible.cfg
  configured module search path = [u'/home/adrian/ansible/my-modules', u'/home/adrian/ansible/lib/modules/']

```

##### SUMMARY
Fixes #20569 (probably)

(module errors if sysctl file uses ';' as comment char instead of  '#').

';' is valid comment char based on http://man7.org/linux/man-pages/man5/sysctl.conf.5.html

anryko reviewed (commented):
There is another place in code where comment check is performed. Check line 323.
I would suggest creating helper static method on the class:
```
    @staticmethod
    def is_comment(line):
        # Lines which begin with a # or ; are considered comments and ignored.
        return line.strip().startswith(('#', ';'))
```
and using it in both cases as ```self.is_comment(line)```

alikins committed:
Handle sysctl.conf files that use ';' for comments

'#' and ';' are both valid comment chars for sysctl.conf files
according to the 'man sysctl.conf':

  "Lines which begin with a # or ; are considered comments and ignored."

Fixes #20569

alikins commented:
updated both places comments are checked. Just used str.startswith()' support for a tuple of prefixes.

anryko commented:
Nice. ```tmpline = line.strip()``` variable is actually never used, must be a leftover code...
